src/current/v25.3/user-defined-functions.md

```diff
@@ -113,6 +113,12 @@ SELECT add(3,5) as sum;
 (1 row)
 ~~~
 
+### Call a UDF from a view
+
+You can include UDF calls in a view definition. Each time the view is queried, the function executes.
```

Comment:
Note that a materialized view will invoke the function only when the view is created/refreshed.